src/bio/bio_xstream.c

```diff
@@ -1242,16 +1233,6 @@ assign_xs_bdev(struct bio_xs_context *ctxt, int tgt_id, enum smd_dev_type st,
 	int			 rc;
 
 	*dev_state = SMD_DEV_NORMAL;
-	if (tgt_id == BIO_SYS_TGT_ID || tgt_id == BIO_STANDALONE_TGT_ID) {
-		d_bdev = find_sys_bdev(dev_type2role(st));
-		if (d_bdev == NULL) {
-			D_ERROR("Failed to find sys bdev\n");
-			return NULL;
-		}
-		/* Assume the sys device in NORMAL state for this moment */
-		return d_bdev;
-	}
-
```

Comment:
Now a device will be assigned to SYS_TGT_ID for RDB (the mapping will be recorded in target table), but I don't think we should treat the SYS_TGT mapping equally with other VOS targets mappings.

Let's take an example, if there is a config having 4 meta SSDs and 3 targets, how should we assign SSDs?

1. Assign 3 SSDs to 3 VOS targets and sys target (sys target share SSD with one of VOS target), leave one SSD unused, or;
2. Assign 1 SSD to sys target, assign the other 3 SSDs to VOS targets;

I think we'd use the 1st policy to assign SSDs, right?

Reply:
sure.